trax/tf_numpy/extensions/extensions.py

```diff
@@ -564,6 +565,93 @@ def tf_dot_general(lhs, rhs, dimension_numbers):
   return tf.einsum(equation, lhs, rhs)
 
 
+# TODO (Zhibo Zhang): Run pylint and complement the docstring.
```

Comment:
Lets do both pylint/docstring changes for both functions this and below before submission. 

Also, I think its better to put the TODOs under your github username instead of your full name (so "DarrenZhang01" instead of "Zhibo Zhang").